src/client.ts

```diff
@@ -181,7 +181,7 @@ export interface ReactSDKClient extends Omit<optimizely.Client, 'createUserConte
 
   fetchQualifiedSegments(options?: optimizely.OptimizelySegmentOption[]): Promise<boolean>;
 
-  getQualifedSegments(): string[] | null;
+  getUserContextInstance(userInfo: UserInfo): optimizely.OptimizelyUserContext | null;
```

Comment:
Not sure but wondering if we can use "getUser" as a pair with existing "setUser" method.

Reply:
Oh. That makes sense. I noticed that the existing `public getUserContextInstance()` wasn't on the interface and only used internally on the concrete implementation, so that's why I chose it.

Do you want me to remove it from the interface and add a `getUser`?

Reply:
Yes, simple getter `getUser() { return this.currentContext }` instead of `getUserContextInstance()` makes more sense to me.

Reply:
Hi @jaeopt,

I've pushed the changes per our call.  Thanks for another review.